Parser/asdl_c.py

```diff
@@ -1194,7 +1194,7 @@ def main(srcfile):
         sys.exit(1)
     if INC_DIR:
         p = "%s/%s-ast.h" % (INC_DIR, mod.name)
-        f = open(p, "wb")
+        f = open(p, "w")
```

Comment:
The "wb" mode is used for disallowing newlines translation. In Python 3 you should use `newline='\n'`.

Reply:
This is Python 2.7.

Reply:
Then why this is changed?

Reply:
Addressed now in 8405be4a2e58bc4d06117da84f581e8514da614c.